src/Models/Constants/Interfaces.ts

```diff
@@ -1,4 +1,4 @@
-import { AbstractMesh, Scene } from '@babylonjs/core';
+import { AbstractMesh, int, Scene } from '@babylonjs/core';
```

Comment:
I don't think this is referenced anywhere in the file. So I don't think we need to add it here?